src/components/listener/dashboard/ListenerDashboard.jsx

```diff
@@ -35,75 +41,163 @@ function ListenerDashboard() {
-                    <hr />
-
-                    <div className='l-menu-items'>
-                        <div className='l-dashbaord-menu-items'>
-                            <FaUser className='menu-icon' />
-                            <h5>Profile</h5>
-                        </div>
-                        <div className='l-dashbaord-menu-items'>
-                            <FontAwesomeIcon icon={faGauge} className='menu-icon' />
-                            <h5>Dashboard</h5>
-                        </div>
-                        <div className='l-dashbaord-menu-items'>
-                            <FaHeart className='menu-icon' />
-                            <h5>Favourite</h5>
+
+            <main>
+                <div className='row l-main-top'>
+                    <div className='col-md-2 l-menu-bar'>
+                        <h4>Menu</h4>
+                        <hr />
+
+                        <div className='l-menu-items'>
+                            <div className='l-dashbaord-menu-items'>
+                                <FaUser className='menu-icon' />
+                                <h5>Profile</h5>
+                            </div>
+                            <div className='l-dashbaord-menu-items'>
+                                <FontAwesomeIcon icon={faGauge} className='menu-icon' />
+                                <h5>Dashboard</h5>
+                            </div>
+                            <div className='l-dashbaord-menu-items'>
+                                <FaHeart className='menu-icon' />
+                                <h5>Favourite</h5>
+                            </div>
+                            <div className='l-dashbaord-menu-items'>
+                                <FaTag className='menu-icon' />
+                                <h5>Tags</h5>
+                            </div>
+                            <div className='l-dashbaord-menu-items'>
+                                <FontAwesomeIcon icon={faUserGroup} className='menu-icon' />
+                                <h5>Friends</h5>
+                            </div>
                         </div>
-                        <div className='l-dashbaord-menu-items'>
-                            <FaTag className='menu-icon' />
-                            <h5>Tags</h5>
+                    </div>
+
+                    <div className='col-md-7 l-discover-genre'>
+                        <div className='l-discover-genre-top'>
+                            <h5>
+                                <FaMusic className='l-discover-genre-music-icon' />
+                            </h5>
+                            <h2>
+                                Discover Genre
+                            </h2>
                         </div>
-                        <div className='l-dashbaord-menu-items'>
-                            <FontAwesomeIcon icon={faUserGroup} className='menu-icon' />
-                            <h5>Friends</h5>
+
```

Comment:
I think it could be good to add a comment here to remember that this is temporary hard coded data. For instance,
`// TODO: replace hard coded genres with top user suggested genres `
and same with the artist and song sections below